themes/src/main/resources/theme/keycloak-preview/account/resources/content.json

```diff
@@ -56,5 +56,15 @@
     "modulePath": "/content/my-resources-page/MyResourcesPage.js",
     "componentName": "MyResourcesPage",
     "hidden": "!features.isMyResourcesEnabled"
+  },
+  {
+    "id": "delete-account",
+    "icon": "pf-icon-disconnected",
+    "path": "delete-account",
+    "label": "deleteAccount",
+    "descriptionLabel": "deleteAccountSummary",
+    "modulePath": "/content/delete-account-page/DeleteAccountPage.js",
+    "componentName": "DeleteAccountPage",
+    "hidden": "!features.deleteAccountAllowed"
```

Comment:
@stianst @zak905  I think I mentioned this before, but I don't like having this feature as part of the left navigation on the main page.  Deleting the account is then just two clicks away.  It should be more buried in the UI so it's a little hard to find and especially hard to execute.

Reply:
@ssilvert looking index.ftl that has been abused quite a bit. Ideally that file should not contain any user information, or sensitive information, and should only contain realm config, not user config, so that it can be cached on the browser side and also doesn't leak anything sensitive about the user since it is not protected.

Reply:
@ssilvert I agree that it may not be the best to have it in the left-hand menu, but where else to put it?

Reply:
@stianst @zak905  The best spot I can think of is to put it on the Personal Info page.  Add a footer to that page and put a link in the bottom right corner.  That way you will only click it if you are looking for it.

Reply:
> @ssilvert looking index.ftl that has been abused quite a bit. Ideally that file should not contain any user information, or sensitive information, and should only contain realm config, not user config, so that it can be cached on the browser side and also doesn't leak anything sensitive about the user since it is not protected.

If that's considered sensitive user information then I agree.  And if we move the link off of the navigation menu then the role check would not need to be in index.ftl.  It could be done much later so that makes it a little cleaner.

Reply:
it can make sens to put it into personal info page as this related to the user account. However, since the account console is meant to be customizable, I guess the user (the keycloak user and not the end user) can also choose where to put it. Based on my experience, I think the account console is not used as-is, but rather customized to match the application look and feel. In my opinion, even if it's easily reachable, the user still needs to login and confirm again, so there are still some steps to go. To sum up we could : 

* add it as footer link in the personal account page
* create a sub-menu for the Personal Account left link and add it as a new item
* add it as a new sub-menu item in the Account Security.

What do you think ?

Reply:
I don't like the second option for several reasons.  One is that it puts the most important screen, Personal Info, in a submenu.

My personal preference from a human factors perspective is to add it as a footer in the personal account page.  You really shouldn't know that this functionality exists unless you go looking for it.

I do understand that with the way the account console is structured, that would be a little odd from a code standpoint.  I think the simplest compromise would be to add it as a sub-menu in Account Security.  So all-in-all, I lean toward the last option.

Reply:
Sounds good. I will post some screenshots in the groups discussion.

Reply:
+1 for putting a link to the Personal Info page. Makes most sense to me.